app/jobs/delius_import_job.rb

```diff
@@ -104,9 +104,35 @@ def process_attachment(attachment_bytes)
 
   def process_decrypted_file(filename)
     Rails.logger.info('[DELIUS] Processing decrypted file')
+    processor = Delius::Processor.new(filename)
```

Comment:
@kjdchapman The `Delius::Processor` class opens a XLSX file and provides an `Enumerable` interface to loop through rows of the spreadsheet. So as for naming, a more descriptive name would probably be `XlsxReader` – it's not unique to Delius, so the module namespacing here is misleading.

It looks like one of the key considerations when originally implementing `Delius::Processor` in 2d07dc7f was that it should stream rows from the XLSX file, rather than attempt to load the entire file into memory. However this seems to be is something that established XLSX-reading gems can do, too ([roo](https://github.com/roo-rb/roo#excel-xlsx-and-xlsm-support), [creek](https://github.com/pythonicrubyist/creek)).

So If I were going for a full refactor, I'd replace this functionality with a gem – I'd rather us not have to be aware of the intricacies of reading XLSX files, nor have to maintain that as part of our codebase. But I didn't need to get too close to it for this PR, so have left it as is.